src/types/rest/response/fills.ts

```diff
@@ -0,0 +1,29 @@
+import { OrderSide } from "./orders";
+
+export interface Fill {
+  fee: number;
+  feeCurrency: string;
+  feeRate: number;
+  future: string;
+  id: string;
+  liquidity: "maker" | "taker";
+  market: string;
+  baseCurrency?: string | null;
+  quoteCurrency: string | null;
+  orderId: number;
+  tradeId: number | null;
+  price: number;
+  side: OrderSide;
+  size: number;
+  time: string
+  type: "orders" | string; // other types undocumented
+}
+
+export type Fills = Fill[];
```

Comment:
Same again, and anywhere else with a "plural" type. Just use `Fill[]` directly where you have an array of the Fill type.